gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy/src/main/resources/schemas/sharedConfiguration/schema-form.json

```diff
@@ -58,6 +58,17 @@
                 }
             }
         },
+        "propagateClientAcceptEncoding":{
+            "title":"Propagate client Accept-Encoding header",
+            "description": "Propagate Accept-Encoding can only be made if \"Enable compression\" is disabled.",
+            "type": "boolean",
+            "gioConfig": {
+                "banner": {
+                    "title": "Propagate client Accept-Encoding header (no decompression if any)",
+                    "text": "The gateway will propagate the Accept-Encoding header's value specified by the client's request to the backend (if any). The gateway will NEVER attempt to decompress the content if the backend response is compressed (gzip, deflate). It is then not possible to apply transformation policy if the body is compressed. Also, body will appear compressed if logging is enabled for the API. DO NOT activate this option if you plan to play with body responses."
```

Comment:
If the client request header includes a value for Accept-Encoding, the gateway will propagate it to the backend.
The gateway will NEVER attempt to decompress the **(request? accept-encoding value?)** content if the backend response is compressed (gzip, deflate). Therefore, transformation policies cannot be applied if the **(request? response?)** body is compressed. If logging is enabled for the API, the **(request? response?)** body will appear compressed. DO NOT activate this option if you plan to interact with body responses.

Can you respond to the questions in **bold**?

Reply:
wololoo I don't know, but it's the text in api v2.

Reply:
If the client request header includes a value for Accept-Encoding, the gateway will propagate it to the backend. The gateway will NEVER attempt to decompress the body content if the backend response is compressed (gzip, deflate). Therefore, transformation policies cannot be applied if the content body is compressed. If logging is enabled for the API, the compressed content body will be logged as is without any transformation applied. DO NOT activate this option if you plan to interact with body responses.

Reply:
:keanu_thanks: